src/Dfc.CourseDirectory.Core/DataStore/Sql/Models/FeChoice.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace Dfc.CourseDirectory.Core.DataStore.Sql.Models
+{
+    public class FeChoice
+    {
+        public Guid Id { get; set; }
+        public int UKPRN { get; set; }
```

Comment:
I'd suggest bolting on `LearnerSatisfaction` and `EmployerSatisfaction` onto the existing `Pttcd.Providers` table

Reply:
Removed and put onto providers now.